src/authV2/azext_authV2/_help.py

```diff
@@ -0,0 +1,113 @@
+
+helps['webapp auth'] = """
+type: group
+short-summary: Manage webapp authentication and authorization in the v2 format.
+"""
+
+helps['webapp auth show'] = """
+type: command
+short-summary: Show the authentication settings for the webapp in the v2 format.
+examples:
+  - name: Show the authentication settings for the webapp. (autogenerated)
+    text: az webapp auth show --name MyWebApp --resource-group MyResourceGroup
+    crafted: true
+"""
+
+helps['webapp auth update'] = """
+type: command
+short-summary: Update the authentication settings for the webapp in the v2 format.
+examples:
+  - name: Update the client ID of the AAD provider already configured
+    text: >
+        az webapp auth update  -g myResourceGroup -n myUniqueApp --set identityProviders.azureActiveDirectory.registration.clientId=ecbacb08-df8b-450d-82b3-3fced03f2b27
+  - name: Enable authentication on the app
+    text: >
+        az webapp auth update -g myResourceGroup -n myUniqueApp --enabled true
+  - name: Pin the runtime version of the app to 1.4.7
+    text: >
+        az webapp auth update -g myResourceGroup -n myUniqueApp --runtime-version 1.4.7
+  - name: Configure the app with file based authentication by setting the config file path
+    text: >
+        az webapp auth update -g myResourceGroup -n myUniqueApp --config-file-path auth.json
+  - name: Configure the app to redirect to AzureActiveDirectory's login page for unauthenticated requests
+    text: >
+        az webapp auth update -g myResourceGroup -n myUniqueApp --unauthenticated-client-action RedirectToLoginPage --redirect-provider AzureActiveDirectory
+  - name: Configure the app to listen to the forward headers X-FORWARDED-HOST and X-FORWARDED-PROTO
+    text: >
+        az webapp auth update -g myResourceGroup -n myUniqueApp --proxy-convention Standard
+"""
+
+helps['webapp auth set'] = """
+type: command
+short-summary: Sets the authentication settings for the webapp in the v2 format.
+examples:
+  - name: Set the json saved in file auth.json as the auth settings for the web app.
+    text: >
+        az webapp auth set -g myResourceGroup -n myUniqueApp --body @auth.json
+"""
+
+helps['webapp auth config-version show'] = """
+type: command
+short-summary: Show the configuration version of the authentication settings for the webapp.
+examples:
+  - name: Show the configuration version of the authentication settings for the webapp (autogenerated)
+    text: >
+        az webapp auth config-version show --name MyWebApp --resource-group MyResourceGroup
+"""
+
+helps['webapp auth config-version revert'] = """
+type: command
+short-summary: Reverts the configuration version of the authentication settings for the webapp from v2 to v1 (classic).
+examples:
+  - name: Revert the configuration version of the authentication settings for the webapp from v2 to v1 (classic) (autogenerated)
+    text: >
+        az webapp auth config-version revert --name MyWebApp --resource-group MyResourceGroup
```

Comment:
I suggest we call this "revertToV1" or "revertToAuthClassic" to make it clear exactly what we're reverting in the name of the command.

Reply:
So I get where you're coming at but is there any requirement that the verb has to be just a verb or can it also have a prep phrase like ToV1 or ToAuthClassic @panchagnula

Reply:
re-discussed this in the easy auth sync, and agreed to leave it at revert.